vendor/github.com/docker/libnetwork/agent.go

```diff
@@ -224,7 +224,7 @@ func (c *controller) agentSetup() error {
 			logrus.Errorf("Error in agentInit : %v", err)
 		} else {
 			c.drvRegistry.WalkDrivers(func(name string, driver driverapi.Driver, capability driverapi.Capability) bool {
-				if capability.DataScope == datastore.GlobalScope {
+				if capability.Multihost {
```

Comment:
No. This is a network driver specific capability, stating whether the networks created by this driver can provide multihost connectivity.

For example, this is the case for macvlan, ipvlan, overlay.

Based on this capability, and the user intention to deploy a swarm network using this driver, libnetwork core will join the cluster wide network DB for this network.